### ISSUE TITLE
[BUG] Mobile App is too eager to present ordered lists

### ISSUE THREAD

**Issue by sandrotanner** [RocketChat/Rocket.Chat.ReactNative#4514]
### Description:

If a user sends messages in the format `X. [TEXT]` (which is used in the german language for dates for example) and puts them on separate lines, the mobile version shows it as a ordered list, while the web version shows it how it was intended.

### Environment Information:

- Rocket.Chat App Version: 4.29.0
- OS Version: tested on latest iPhone and Android versions

### Steps to reproduce:

1. Write the following test message:
```
Lorem Ipsum
6. Oktober: blah
24. Oktober: blah
25. Oktober: blah
Lorem Ipsum
```

2. View it in the app versus the browser

### Expected behavior:

The actual text is displayed on all devices

### Actual behavior:
On phones the following is shown:
```
Lorem Ipsum
1. Oktober: blah
2. Oktober: blah
3. Oktober: blah
Lorem Ipsum
```

Screenshot: https://snipboard.io/ivuoZM.jpg

When copying the message the original text is copied, its only the displayed text that is wrong.

### Additional context:

This issue is problematic for regions where a DMY time format is used and the delimiter "." is used (see https://en.wikipedia.org/wiki/Date_format_by_country). When no using the dot, it does not happen.

**Comment by GleidsonDaniel** [comment]
It has already been resolved in version 4.30.0, it will be released this week.

https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/4388

**Comment by sandrotanner** [comment]
Thank you! @GleidsonDaniel
